arangod/RestHandler/RestReplicationHandler.h

```diff
@@ -25,11 +25,11 @@
 #ifndef ARANGOD_REST_HANDLER_REST_REPLICATION_HANDLER_H
 #define ARANGOD_REST_HANDLER_REST_REPLICATION_HANDLER_H 1
 
+#include "Aql/types.h"
```

Comment:
It seems somewhat unnecessary to include Aql/types.h here, though it was included before as well.
Can it be removed?

Reply:
`aql::QueryId` is used in this file. And as that's a `typedef` I cannot forward-declare it, and would rather not redeclare it here.